bindings/python/iota_sdk/types/context_input.py

```diff
@@ -22,11 +23,31 @@ class ContextInput():
     type: int
 
 
+@json
+@dataclass
+class CommitmentContextInput(ContextInput):
+    """A Commitment Input allows referencing a commitment to a certain slot.
+
+    Attributes:
+        type: The type of context input.
+        commitment_id: The commitment identifier to reference to.
+    """
+    commitment_id: HexStr
+    type: int = field(
+        default_factory=lambda: int(
+            ContextInputType.Commitment),
+        init=False)
+
+
 @json
 @dataclass
 class BlockIssuanceCreditContextInput(ContextInput):
     """A Block Issuance Credit (BIC) Context Input provides the VM with context for the value of
     the BIC vector of a specific slot.
+
+    Attributes:
+        type: The type of context input.
+        account_id: The BIC of an account to use.
```

Comment:
Huh?

Reply:
huh :D

Reply:
This comment doesn't make any sense 😆

Reply:
Yeah should just be ID

Reply:
well its like this in the tip it seems.. hmm  

Made a comment, problem for another day.
https://github.com/iotaledger/tips-draft/pull/19/files#r1325846278